File: arm.64/frame/blazor/src/trip.cs
using System.Collections.Generic;

namespace Pat {
    public static class Tripper {
        public static List<Triplet> Trip() {
            return new List<Triplet> {
                new Triplet("/version", "{}", "{\"once\":true}"),
                new Triplet("/storage", "{}", "{\"once\":true}"),
                new Triplet("sql.help", "{}", "{\"once\":true}")
            };
        }
    }
}

File: arm.64/frame/avalonia/src/trip.fs
namespace Pat

module Tripper =
    let trip () : Triplet list = [
        { Address = "/version"; Payload = "{}"; Options = "{\"once\":true}" }
        { Address = "/storage"; Payload = "{}"; Options = "{\"once\":true}" }
        { Address = "sql.help"; Payload = "{}"; Options = "{\"once\":true}" }
    ]
